### DISCUSSION TITLE
Favorites customization in profiles

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#597] [Q&A] [answered]
A popular request is to differentiate some of the properties of favorites in profiles.

Such as different visibility, use in navigation, etc. in each profile.

This applies to all types: favorites, routes, tracks / areas.

**Comment by devemux86** [comment]
This combined with the [no-go areas in favorites](https://github.com/devemux86/cruiser/discussions/545) solves the [request](https://github.com/devemux86/cruiser/discussions/506#discussioncomment-10045627) by @WolfgangZ42. :slightly_smiling_face: 

And allows the use of favorites as map layers with different on / off states in each profile.

**Comment by devemux86** [comment]
Favorites, routes, tracks / areas visibility (and use in navigation) can now be customized separately per app profile:

- https://github.com/devemux86/cruiser/discussions/610

**Comment by devemux86** [comment, marked as answer]
- Favorites, routes, tracks / areas visibility (and use in navigation)
can be customized separately per profile in https://github.com/devemux86/cruiser/discussions/641

**Reply by WolfgangZ42** [reply]
It doesn't work anymore to set the function (use in navigation) of a polygon no-go-area profile related. I'm quite sure it worked already. 
You can set the visibility of such an area profile related, but not the functionality. 
In my case the Diesel-Ban in Munich isn't relevant for motorcycles. If I clear the no-go-area item in My_collection of the Motorcycle profile it also vanished in the Car profile - vice versa. 
Don't know in which version of Cruiser this happened, I very seldom ride through Munich on motorcycle. 

Unfortunately a similar problem with radial no-go-areas in favorites: the folder "No-go areas" contains two areas, one for the Car profile and the other for Motorcycle tours. 
<img width="200" src="https://github.com/user-attachments/assets/a5306ac1-817e-489e-86c3-668d91788dd4"/>. . . .<img width="200" src="https://github.com/user-attachments/assets/c5340ff4-b676-4466-917f-c12f55e8d7bc"/>
There is no option to differentiate between the profiles - the “Delete” option applies to all profiles.

**Reply by devemux86** [reply]
@WolfgangZ42 Thanks for the report.

> It doesn't work anymore to set the function (use in navigation) of a polygon no-go-area profile related.

"Use in navigation" (for nearby favorites) only exists in "Favorites" (not no-go areas) and is saved per profile.

The no-go flag in "Tracks / Areas" was never saved per profile. Do you mean that?

> the folder "No-go areas" contains two areas, one for the Car profile and the other for Motorcycle tours.

Favorites are organized in groups with common properties.
To set different properties per profile, store them in different groups.

**Reply by devemux86** [reply]
- **Tracks / Areas**
The no-go flag for the "Tracks / Areas" will be stored per profile in the next update.

- **Favorites**
For the "Favorites" this function already works. The no-go flag is set in group creation.
So if want to use different no-go areas from "Favorites" in profiles,
use different no-go groups and use the "Show on map" to change their visibility per profile.

**Reply by devemux86** [reply]
@WolfgangZ42 

Can you please verify that everything works correctly with "Favorites"?
Visibility and use in navigation.
For "No-go areas" in favorites, need to create different groups per profile.

"Tracks / Areas" no-go flag per profile will be included in the next update.

**Reply by WolfgangZ42** [reply]
> will be stored per profile in the next update

Thanks! 🙂 

> The no-go flag is set in group creation

That's thru - **again what learned!** Now I know, that more then one "No-go areas"-group is possible, that I can store NGAs into different groups at creation and even can change the group later. "Show on map" works perfectly well. Again: Thanks! 🙂

**Reply by devemux86** [reply]
> Tracks no-go flag per profile will be included in the next update.

Implemented in **[Cruiser 5.1.0-b4](https://github.com/devemux86/cruiser/discussions/671)**.

**Reply by devemux86** [reply]
> Tracks no-go flag per profile will be included in the next update.

- Implemented in https://github.com/devemux86/cruiser/discussions/684
